### PR TITLE
Add a coding standards GA to fix code with Laravel Pint.

### PR DESCRIPTION
**Pull request by lucasmichot**
This PR adds a coding standards GA to fix code with Laravel [Pint](https://laravel.com/docs/10.x/pint).

It also fixes the code to match those coding standards

### CODE REVIEW

File: src/Acclaim/Provider.php [hunk 1]
```diff
@@ -51,11 +51,11 @@ protected function getUserByToken($token)
     protected function mapUserToObject(array $user)
     {
         return (new User())->setRaw($user)->map([
-            'id'       => $user['data']['id'],
+            'id' => $user['data']['id'],
             'nickname' => $user['data']['first_name'],
-            'name'     => $user['data']['first_name'],
```
Comment (atymic) [inline src/Acclaim/Provider.php:56]: Could we leave the align arrays rule on? changes such as these just pollute the commit history imo.

File: .github/workflows/coding-standards.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+name: Fix code styling
+
+on:
+  workflow_dispatch:
```
Comment (atymic) [inline .github/workflows/coding-standards.yml:4]: Should this also not run on commit to the master branch as well?

Comment (lucasmichot) [reply .github/workflows/coding-standards.yml:4]: I would probably stick to a dispatch, to mimic 
https://github.com/laravel/.github/blob/main/.github/workflows/coding-standards.yml

File: src/Acclaim/Provider.php (hunk 1 above)
Comment (lucasmichot) [reply src/Acclaim/Provider.php]: Sure 👍

File: .github/workflows/coding-standards.yml (hunk 2 above)
Comment (atymic) [reply .github/workflows/coding-standards.yml:4]: I mean, just the lint (not the fix)

**Comment by atymic** [comment]
Could you resolve the conflicts?

**Comment by lucasmichot** [comment]
> Could you resolve the conflicts?

Conflicts resolved

File: .github/workflows/coding-standards.yml (hunk 2 above)
Comment (lucasmichot) [reply .github/workflows/coding-standards.yml:4]: I see; in any case, if lint should format for workflow_dispatch and lint on push, we need 2 different workflows files.
I propose to merge this one, and I'll make a PR for the linting part on push into master

WDYT ?

Comment (atymic) [reply .github/workflows/coding-standards.yml:4]: Can we have a lint workflow for PRs as well please?

**Pull request merged**